index.js

```diff
@@ -54,15 +55,21 @@ class HRPCServer extends EventEmitter {
 }
 
 module.exports = class HRPC extends EventEmitter {
-  static createServer (onclient) {
-    return new HRPCServer(this, onclient)
+  static createServer (src, onclient = src) {
```

Comment:
If you don't specify `onclient`, but `src` is an `EventEmitter`, won't this throw in the `this.on('client')` line?

Reply:
fixed in 2.1.1